internal/services/subscription/subscription_resource.go

```diff
@@ -281,13 +281,12 @@ func resourceSubscriptionUpdate(d *pluginsdk.ResourceData, meta interface{}) err
 	if d.HasChange("tags") {
 		tagsClient := meta.(*clients.Client).Resource.TagsClientForSubscription(subscriptionId.ID())
 		t := tags.Expand(d.Get("tags").(map[string]interface{}))
-		scope := fmt.Sprintf("subscriptions/%s", *subscriptionId)
 		tagsResource := resources.TagsResource{
 			Properties: &resources.Tags{
 				Tags: t,
 			},
 		}
-		if _, err = tagsClient.CreateOrUpdateAtScope(ctx, scope, tagsResource); err != nil {
+		if _, err = tagsClient.CreateOrUpdateAtScope(ctx, subscriptionId.ID(), tagsResource); err != nil {
```

Comment:
this would change the first parameter from `subscriptions/XXXX` to `/subscriptions/XXXX` - meaning this'll be sent over the wire as `//subscriptions` rather than `/subscriptions` (previously) - presumably there's a test covering updating the tags?

Reply:
I did test updating the tags and the server side can correctly handle `//subscriptions` case. However, since // is not a standard way for a URL, I reverted it. I have also updated the update testcase.